blocks/init/src/blocks/custom/featured-posts/featured-posts.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Template for the Featured Posts view.
+ *
+ * @since 1.0.0
+ * @package Eightshift_Boilerplate\Blocks.
+ */
+
+$block_class = $attributes['blockClass'] ?? '';
+$posts  = $attributes['posts'] ?? '';
```

Comment:
create an condition if posts don't exist to exit before the execution

`if(!$posts){
return
}`